dashboard/app/views/levels/_dialogHelper.js

```diff
@@ -49,7 +49,7 @@
     dialog.show();
   }
 
-  window.showStartOverDialog = function(callback) {
+  appOptions.dialog.showStartOverDialog = function(callback) {
```

Comment:
Also potentially confusing is that elsewhere we have appOptions.Dialog to pass our dialog class.